src/Jobs/Recruit.Vacancies.Jobs/Triggers/QueueTriggers/DeleteStaleQueryStoreDocumentsQueueTrigger.cs

```diff
@@ -59,8 +59,7 @@ public async Task DeleteStaleQueryStoreDocumentsAsync([QueueTrigger(QueueNames.D
 
                 var documentTypesToDelete = new [] 
                 {
-                    nameof(EditVacancyInfo), 
-                    nameof(ProviderEditVacancyInfo),
+                    "EditVacancyInfo", 
```

Comment:
any reason for not continuing the theme of nameof() ?

Reply:
There was fault (harmless) in the original code. EmployerEditVacancyInfo (formerly EditVacancyInfo) and ProviderEditVacancyInfo both have viewType of `EditVacancyInfo` and hence `nameof(ProviderEditVacancyInfo)` was not adding any value. Since I renamed the `EditVacancyInfo` to `EmployerEditVacancyInfo` I am unable to use `nameof` as I want to match `EditVacancyInfo` and not `EmployerEditVacancyInfo` or `ProviderEditVacancyInfo`. Hope this makes sense.